plugins/importexport/doaj/filter/DOAJJsonFilter.inc.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+/**
+ * @file plugins/importexport/doaj/filter/DOAJJsonFilter.inc.php
+ *
+ * Copyright (c) 2014-2016 Simon Fraser University Library
+ * Copyright (c) 2000-2016 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class DOAJJsonFilter
+ * @ingroup plugins_importexport_doaj
+ *
+ * @brief Class that converts an Article to a DOAJ JSON string.
+ */
+
+import('lib.pkp.plugins.importexport.native.filter.NativeImportExportFilter');
+
+
+class DOAJJsonFilter extends NativeImportExportFilter {
+	/**
+	 * Constructor
+	 * @param $filterGroup FilterGroup
+	 */
+	function __construct($filterGroup) {
+		$this->setDisplayName('DOAJ JSON export');
```

Comment:
Hmm, I remember that now, thanks. I haven't looked into it in detail. Let's keep it consistent for now.